src/lib/services/tx/lcd.ts

```diff
@@ -9,28 +13,45 @@ export const getTxDataLcd = async (endpoint: string, txHash: string) =>
     .get(`${endpoint}/cosmos/tx/v1beta1/txs/${encodeURI(txHash)}`)
     .then(({ data }) => parseWithError(zTxByHashResponseLcd, data));
 
+export const getTxsByHashLcd = async (endpoint: string, txHash: string) =>
```

Comment:
is this the same as `getTxDataLcd`

Reply:
I'd prefer `getTxsByHashLcd` to return `Transaction[]` instead.